model/AI/Classes/AIPackage.md

```diff
@@ -0,0 +1,107 @@
+-------
+SPDX-License-Identifier: Community-Spec-1.0
+
+# AIPackage
+
+## Description
+
+Metadata information that can be added to a package to describe an AI application or trained AI model. 
+
+## Metadata
+
+- SubclassOf: Software/Package
+						Defaults are: 0..*
+## Properties
+-energyConsumption
+  - type: xsd:string
+  - minCount: 0
+  - maxCount: 1
+  
```

Comment:
this and the following newlines should be dropped. The parsing is rather fragile and expects very specific structure

Reply:
Thanks. Fixed per your suggestion